deployment/keystone/deploy.go

```diff
@@ -271,24 +267,33 @@ func ConfigureRegistry(ctx context.Context, lggr logger.Logger, req ConfigureCon
 
 	// TODO: annotate nodes with node_operator_id in JD?
 
+	donsToRegister := []DONToRegister{}
+	for name, nodes := range donToNodes {
+		donsToRegister = append(donsToRegister, DONToRegister{
+			Name:  name,
+			F:     uint8(len(nodes) / 3), // TODO: incorrect, should come from config
```

Comment:
is it straightforward to fix this now , eg add  map[donName] -> F to ConfigureContractsRequest or something similar?

Reply:
and for compatibilty, if no value then default to existing logic